### PR TITLE
[SEMI-MODULAR] Nerve staple bounty

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

Nerve staple bounty

Adds a -10 value quirk which applies an item that pacifies you (-8) and permanently takes up the glasses slot (-2). **Can't be taken with nearsighted or pacifist quirks**.

Also fixes the ghost cafe interaction with quirks so that things spawn properly.

## How This Contributes To The Skyrat Roleplay Experience

bounty + bug fix

## Changelog

:cl:
add: Adds the nerve staple, a nodrop item that pacifies you, and the corresponding quirk
/:cl:

### CODE REVIEW

**Comment by Sonic121x** [comment]
can it be get rid of with the HARS mutation?(datum/mutation/human/headless)
or by amputate the head and reconnect it.

File: code/modules/awaymissions/corpse.dm
Comment (tf-4) [inline code/modules/awaymissions/corpse.dm]: skyrat edit

**Comment by SpaceVampire** [comment]
If you can just remove the item and the effect stops, I don't know if it is really a good replacement for those quirks at the exact same price. Given that there is normally no real way to deal with any of the other quirks. You can't just get brain surgery in the middle of the round to fix them or anything.

**Comment by tf-4** [comment]
Please don't bundle fixes in with features

**Comment by Gandalf2k15** [comment]
Make it so that if you remove it, they die

File: code/modules/awaymissions/corpse.dm
Comment (Seris02) [reply code/modules/awaymissions/corpse.dm]: doesn't need one, it's already inside a skyrat edit

File: code/modules/awaymissions/corpse.dm
Comment (jjpark-kb) [inline code/modules/awaymissions/corpse.dm]: needs to have documentation ///

File: modular_skyrat/master_files/code/datums/quirks/negative.dm
Comment (jjpark-kb) [inline modular_skyrat/master_files/code/datums/quirks/negative.dm:11]: needs to have documentation ///

File: modular_skyrat/master_files/code/datums/quirks/neutral.dm
Comment (jjpark-kb) [inline modular_skyrat/master_files/code/datums/quirks/neutral.dm]: whats the difference between on_equip_items and on_equip_item... why would you rather have the singular and put a loop when I assume the plural does that.

File: modular_skyrat/master_files/code/datums/quirks/neutral.dm
Comment (jjpark-kb) [inline modular_skyrat/master_files/code/datums/quirks/neutral.dm]: this does not seem related to nerve staples. atomize.

File: modular_skyrat/master_files/code/datums/quirks/neutral.dm
Comment (jjpark-kb) [inline modular_skyrat/master_files/code/datums/quirks/neutral.dm]: this does not seem related to nerve staples. atomize.

**Review by jjpark-kb — CHANGES_REQUESTED** [review]
deal with what I mentioned in order to gain approval.

**Comment by Seris02** [comment]
do not merge until #10010 is merged and after that I'll fix up the code to reflect that PR

**Comment by tf-4** [comment]
fix changelog pls

File: modular_skyrat/master_files/code/datums/quirks/negative.dm
Comment (tf-4) [inline modular_skyrat/master_files/code/datums/quirks/negative.dm]:
```suggestion
	gain_text = span_danger("You suddenly can't raise a hand to hurt others!")
	lose_text = span_notice("You think you can defend yourself again.")
```
Yes, this does work

**Review by tf-4 — COMMENTED** [review]
Final change then ready to go

File: modular_skyrat/master_files/code/datums/quirks/negative.dm
Comment (Fikou) [inline modular_skyrat/master_files/code/datums/quirks/negative.dm]: this makes it sound like your face is being melted

File: modular_skyrat/master_files/code/modules/clothing/glasses/nerve_staple.dm
Comment (Fikou) [inline modular_skyrat/master_files/code/modules/clothing/glasses/nerve_staple.dm:1]: make it acid/fireproof?

**Review by tf-4 — APPROVED** [review]
Seems good

**Pull request merged**
